scripts/build.sh

```diff
@@ -14,7 +14,7 @@ echo "***** Install constructor *****"
 mamba install --yes \
     --channel conda-forge --override-channels \
     jinja2 curl libarchive \
-    "constructor>=3.9.3"
+    "constructor>=3.11.1"
```

Comment:
How about this, @jaimergp?
```suggestion
    "constructor==3.11.1"
```

Reply:
No, in this case we don't need to pin `constructor`. The only thing we need to ensure is that we reliably ship the same packages in the installers (hence the other comment before).